integration/tests/economy/conftest.py

```diff
@@ -19,14 +18,6 @@ def sol_client_tx_v2(pytestconfig: Config):
     return client
 
 
-@pytest.fixture(scope="class")
-def counter_contract(account_with_all_tokens, client_and_price, web3_client_sol, web3_client):
```

Comment:
for operator tests it is important to have counter_contract as it was (with client_and_price)

Reply:
Reverted ✅